produced with commands (and yes, there's space):

(cd ../../SOURCES; cvs up rpm.groups)
awk -vq="'" '/^[A-Z].*/ { printf("    %c%s%c,\n", q, $0, q) }' < ../../SOURCES/rpm.groups > groups

and then inserted in vim:
:r groups

--- rpmlint-0.71/TagsCheck.py~	2005-07-01 05:59:46.000000000 +0300
+++ rpmlint-0.71/TagsCheck.py	2006-02-28 23:51:27.000000000 +0200
@@ -15,92 +15,95 @@
 import Config
 
 DEFAULT_VALID_GROUPS=(
-    'Accessibility',
-    'Archiving/Backup',
-    'Archiving/Cd burning',
-    'Archiving/Compression',
-    'Archiving/Other',
-    'Books/Computer books',
-    'Books/Faqs',
-    'Books/Howtos',
-    'Books/Literature',
-    'Books/Other',
-    'Communications',
-    'Databases',
-    'Development/C',
-    'Development/C++',
-    'Development/Databases',
-    'Development/GNOME and GTK+',
-    'Development/Java',
-    'Development/KDE and Qt',
-    'Development/Kernel',
-    'Development/Other',
-    'Development/Perl',
-    'Development/Python',
-    'Editors',
-    'Education',
-    'Emulators',
-    'File tools',
-    'Games/Adventure',
-    'Games/Arcade',
-    'Games/Boards',
-    'Games/Cards',
-    'Games/Other',
-    'Games/Puzzles',
-    'Games/Sports',
-    'Games/Strategy',
-    'Graphical desktop/Enlightenment',
-    'Graphical desktop/FVWM based',
-    'Graphical desktop/GNOME',
-    'Graphical desktop/Icewm',
-    'Graphical desktop/KDE',
-    'Graphical desktop/Other',
-    'Graphical desktop/Sawfish',
-    'Graphical desktop/WindowMaker',
-    'Graphical desktop/Xfce',
-    'Graphics',
-    'Monitoring',
-    'Networking/Chat',
-    'Networking/File transfer',
-    'Networking/IRC',
-    'Networking/Instant messaging',
-    'Networking/Mail',
-    'Networking/News',
-    'Networking/Other',
-    'Networking/Remote access',
-    'Networking/WWW',
-    'Office',
-    'Publishing',
-    'Sciences/Astronomy',
-    'Sciences/Biology',
-    'Sciences/Chemistry',
-    'Sciences/Computer science',
-    'Sciences/Geosciences',
-    'Sciences/Mathematics',
-    'Sciences/Other',
-    'Sciences/Physics',
-    'Shells',
-    'Sound',
-    'System/Base',
-    'System/Configuration/Boot and Init',
-    'System/Configuration/Hardware',
-    'System/Configuration/Networking',
-    'System/Configuration/Other',
-    'System/Configuration/Packaging',
-    'System/Configuration/Printing',
-    'System/Fonts/Console',
-    'System/Fonts/True type',
-    'System/Fonts/Type1',
-    'System/Fonts/X11 bitmap',
-    'System/Internationalization',
-    'System/Kernel and hardware',
-    'System/Libraries',
-    'System/Servers',
-    'System/X11',
-    'Terminals',
-    'Text tools',
-    'Toys',
-    'Video',
+    'Applications',
+    'Applications/Archiving',
+    'Applications/Communications',
+    'Applications/Console',
+    'Applications/Databases',
+    'Applications/Databases/Interfaces',
+    'Applications/Dictionaries',
+    'Applications/Editors',
+    'Applications/Editors/Emacs',
+    'Applications/Editors/Vim',
+    'Applications/Editors/XEmacs',
+    'Applications/Emulators',
+    'Applications/Engineering',
+    'Applications/File',
+    'Applications/Graphics',
+    'Applications/Games',
+    'Applications/Networking',
+    'Applications/Mail',
+    'Applications/Math',
+    'Applications/Multimedia',
+    'Applications/News',
+    'Applications/Printing',
+    'Applications/Publishing',
+    'Applications/Publishing/SGML',
+    'Applications/Publishing/TeX',
+    'Applications/Publishing/XML',
+    'Applications/Publishing/XML/Java',
+    'Applications/Science',
+    'Applications/Shells',
+    'Applications/Sound',
+    'Applications/Spreadsheets',
+    'Applications/System',
+    'Applications/Terminal',
+    'Applications/Text',
+    'Applications/WWW',
+    'Base',
+    'Base/Kernel',
+    'Base/Authentication and Authorization',
+    'Base/Utilities',
+    'Daemons',
+    'Development',
+    'Development/Building',
+    'Development/Debuggers',
+    'Development/Languages',
+    'Development/Languages/Fortran',
+    'Development/Languages/Java',
+    'Development/Languages/Modula3',
+    'Development/Languages/PHP',
+    'Development/Languages/Perl',
+    'Development/Languages/Python',
+    'Development/Languages/Scheme',
+    'Development/Languages/Tcl',
+    'Development/Libraries',
+    'Development/Libraries/Libc',
+    'Development/Tools',
+    'Development/Version Control',
+    'Documentation',
+    'Fonts',
+    'Libraries',
+    'Libraries/Python',
+    'Libraries/XML',
+    'Networking',
+    'Networking/Admin',
+    'Networking/Daemons',
+    'Networking/Daemons/Java',
+    'Networking/Daemons/Java/Extensions',
+    'Networking/Daemons/Java/Servlets',
+    'Networking/News',
+    'Networking/Utilities',
+    'Themes',
+    'Themes/GTK+',
+    'X11',
+    'X11/Amusements',
+    'X11/Applications',
+    'X11/Applications/Editors',
+    'X11/Applications/Games',
+    'X11/Applications/Games/Strategy',
+    'X11/Applications/Graphics',
+    'X11/Applications/Multimedia',
+    'X11/Applications/Networking',
+    'X11/Applications/Publishing',
+    'X11/Applications/Science',
+    'X11/Applications/Sound',
+    'X11/Development/Libraries',
+    'X11/Development/Tools',
+    'X11/Libraries',
+    'X11/Servers',
+    'X11/Window Managers',
+    'X11/Window Managers/Tools',
     )
 
 # liste grabbed from www.opensource.org/licenses
